<ui version="4.0" >
 <author>Peter Zhou and Ian Monroe</author>
 <class>ScriptManagerBase</class>
 <widget class="QWidget" name="ScriptManagerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>517</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>450</width>
    <height>500</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Installed Amarok Scripts:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ScriptSelector" native="1" name="pluginWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="kcfg_AutoUpdateScripts">
     <property name="toolTip">
      <string>Automatically download and install updates for the built-in scripts from the Amarok website if available</string>
     </property>
     <property name="whatsThis">
      <string>Automatically download and install updates for the built-in scripts from the Amarok website if available</string>
     </property>
     <property name="text">
      <string>Automatically update built-in scripts</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QPushButton" name="installButton" >
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Install Script</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="uninstallButton" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Uninstall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="retrieveButton" >
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Get More Scripts</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0" />
 <layoutfunction spacing="KDialog::spacingHint" />
 <customwidgets>
  <customwidget>
   <class>ScriptSelector</class>
   <extends>QWidget</extends>
   <header>ScriptSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
